temp-storage/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "fs": "0.0.1-security",
```

Comment:
These should all be removed because they are built-in dependencies. I'm surprised these can even be installed. any module that can be found on https://nodejs.org/api/index.html is built-in

Reply:
removed them as well :)